dcos_installer/upgrades.py

```diff
@@ -0,0 +1,113 @@
+'''
```

Comment:
nit: I think `upgrade.py` would be more consistent with the other module names in this directory.